priv/erlang.config.in

```diff
@@ -31,8 +31,8 @@
  {lager,
   [{handlers,
     [
-      {lager_console_backend, info},
-      {lager_file_backend, [{file, "priv/log/error.log"},   {level, error}]},
+      {lager_console_backend, [info, {lager_default_formatter, [time, " [", severity, "] ", {site, [site, " "], ""}, {module, [module, ":", line, " "], ""}, message, "\n"]}]},
+      {lager_file_backend, [{file, "priv/log/error.log"}, {level, error}, {formatter, lager_default_formatter}]},
```

Comment:
Todo: check how this influences `error.log` output. We want lager metadata in `error|console.log` too.